src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -93,12 +94,16 @@ const CrashHeader = createReactClass({
 
   render() {
     let {stackView, stackType, newestFirst} = this.props;
+    let crashTitle = this.props.title !== undefined ? this.props.title : t('Exception');
 
     return (
       <div className="crash-title">
         {this.props.beforeTitle}
+        {crashTitle === 'Exception' ? (
```

Comment:
that'll break in another language

Reply:
☝️ Actually maybe not. @billyvg You know how translations work?

Reply:
I believe it returns the string that you used